cli/azd/cmd/env.go

```diff
@@ -261,8 +264,13 @@ func (en *envNewAction) Run(ctx context.Context) error {
 		return err
 	}
 
+	environmentName := ""
+	if len(en.args) >= 1 {
+		environmentName = en.args[0]
+	}
+
```

Comment:
nit (corner cases:)

Do we want to log a warning if len(en.args) > 1 ?  Something like "all arguments after name are ignored".
Or should we fail saying, "expecting max arguments equals to 1"

Same questions for using `-e name` .  Should we add warning or fail?

Reply:
Add `MaximumNArgs` validation, which will print an error like: `accepts at most 1 arg(s), received 2`

Reply:
For `-e name`, let's wait until we get user feedback or develop further thinking about `azd`'s environment usage. We have the same problems with `env list` and `env select`. I think a warning could be useful or removing the flag entirely. I'm not sure about a clear preference yet.